docs/config.md

```diff
@@ -39,6 +39,9 @@ The following fields are supported:
     * Path to rootfs to test against
     * If a relative path is provided, it will be interpreted as relative to
       `vmtest.toml`
+* `arch` (string)
+    * Default: the architecture vmtest was built for.
```

Comment:
I think this should work with images too. I don't have any images from a foreign arch to validate though.